### PR TITLE
Sy 1461 fix issues with improper window shutdown throwing errors

### PR DESCRIPTION
**Pull request by emilbon99**
# Feature Pull Request Template

## Key Information

- **Linear Issue**: [SY-#](https://linear.app/synnax/issue/)

## Description

Please write a short (2-3 sentence) description describing the changes.

## Basic Readiness

- [ ] I have performed a self-review of my code.
- [ ] I have added relevant tests to cover the changes to CI.
- [ ] I have needed QA steps to the [release
      candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template that cover these changes.
- [ ] I have updated in-code documentation to reflect the changes.
- [ ] I have updated user-facing documentation to reflect the changes.

## Backwards Compatibility

The following makes sure that this feature does not break backwards compatability.

### Data Structures

- [ ] Server - I have ensured that previous versions of stored data structures are
      properly migrated to new formats.
- [ ] Console - I have ensured that previous versions of stored data structures are
      properly migrated to new formats.

### API Changes

- [ ] Server - The server API is backwards-compatible
- The following client APIs are backwards-compatible:
  - [ ] C++
  - [ ] TypeScript
  - [ ] Python

### Breaking Changes

If anything in this section is not true, please list all breaking changes.

### CODE REVIEW

File: console/src/layout/slice.ts [hunk 1]
```diff
@@ -249,7 +249,10 @@ export const { actions, reducer } = createSlice({
     remove: (state, { payload: { keys } }: PayloadAction<RemovePayload>) => {
       keys.forEach((contentKey) => {
         const layout = select(state, contentKey);
-        if (layout == null) return;
+        if (layout == null || layout.key == MAIN_WINDOW) {
+          console.log("MAIN WINDOW - NOT CLOSING SLICE");
```
Comment (pjdotson) [inline console/src/layout/slice.ts:253]: Doesn't really make sense to log this message when `layout == null`

File: console/src/layout/Window.tsx [hunk 2]
```diff
@@ -112,6 +116,7 @@ const WindowInternal = (): ReactElement | null => {
       ctx?.start(Haul.ZERO_ITEM, [Haul.FILE]);
   };
 
+  console.log(layout, win);
```
Comment (pjdotson) [inline console/src/layout/Window.tsx:119]: Remove

File: console/src-tauri/gen/schemas/windows-schema.json
Comment (pjdotson) [inline console/src-tauri/gen/schemas/windows-schema.json:1]: Same comment - why did this file get changed?

**Review by pjdotson — APPROVED** [review]
Approved after comments addressed.

File: console/src-tauri/gen/schemas/windows-schema.json
Comment (emilbon99) [reply console/src-tauri/gen/schemas/windows-schema.json:1]: Rolled back

File: console/src/layout/Window.tsx (hunk 2 above)
Comment (emilbon99) [reply console/src/layout/Window.tsx]: Removed

File: console/src/layout/slice.ts (hunk 1 above)
Comment (emilbon99) [reply console/src/layout/slice.ts]: Removed

**Pull request merged**